src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -740,6 +740,7 @@
       {% set attributes = form.vars.attr %}
       {{ attributes.placeholder is defined ?  attributes.placeholder : 'Choose file(s)'|trans({}, 'Admin.Actions') }}
     </label>
+    {{- block('custom_file_after') -}}
```

Comment:
I wonder if you really need this block here 🤔
I have the feeling that they are actually not related with the one `downloadable_file_widget` since it's a différent widget type And that `downloadable_file_widget` could work as expected independently without this new block